src/wp-includes/category-template.php

```diff
@@ -1084,10 +1084,10 @@ function _wp_object_name_sort_cb( $a, $b ) {
  *
  * @param object $a The first object to compare.
  * @param object $b The second object to compare.
- * @return bool Whether the count value for `$a` is greater than the count value for `$b`.
+ * @return int the count value for `$a` minus the count value for `$b` (less than, equal to, or greater than zero).
```

Comment:
Should update return doc.

```suggestion
 * @return int Whether the count value for `$a` minus the count value for `$b` (less than, equal to, or greater than zero).
```

Reply:
I don't think this suggestion is correct.

The return type is `int`, and can be "less than, equal to, or greater than zero". Adding "Whether" here suggests a `bool` or `1|0` return value, which isn't the case here.